ServiceConfig.cs

```diff
@@ -25,13 +28,25 @@ public ServiceConfig(string filename)
             // Read credentials from envrionment
             RepositoryId = Environment.GetEnvironmentVariable("REPOSITORY_ID");
             ServicePrincipalKey = Environment.GetEnvironmentVariable("SERVICE_PRINCIPAL_KEY");
+
             var base64EncodedAccessKey = Environment.GetEnvironmentVariable("ACCESS_KEY");
             if (base64EncodedAccessKey == null)
             {
                 throw new InvalidOperationException("Cannot continue due to missing access key.");
```

Comment:
Shouldn't throw the exception anymore when there is no access key environment variable since it isn't required when using LfdsUsernamePassword